Core.DomainServices/Queries/SystemUsage/QueryReadModelsWhichShouldExpire.cs

```diff
@@ -25,7 +25,23 @@ public IQueryable<ItSystemUsageOverviewReadModel> Apply(IQueryable<ItSystemUsage
                     // Expiration data defined
                     x.SourceEntity.ExpirationDate != null &&
                     // Expiration date has passed
-                    x.SourceEntity.ExpirationDate < currentTime
+                    x.SourceEntity.ExpirationDate < currentTime &&
```

Comment:
contract conditions are not combined with other conditions to make the system invalid... if any of the date related conditions will make it invalid and it is currently active, THEN it is included